app/src/main/java/nyc/c4q/jonathancolon/inContaq/utlities/graphs/DailyGraph.java

```diff
@@ -0,0 +1,216 @@
+package nyc.c4q.jonathancolon.inContaq.utlities.graphs;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.animation.BounceInterpolator;
+
+import com.db.chart.Tools;
+import com.db.chart.animation.Animation;
+import com.db.chart.model.LineSet;
+import com.db.chart.view.LineChartView;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.concurrent.ExecutionException;
+
+import nyc.c4q.jonathancolon.inContaq.utlities.dailysms.DailyReceivedWorkerTask;
+import nyc.c4q.jonathancolon.inContaq.utlities.dailysms.DailySentWorkerTask;
+import nyc.c4q.jonathancolon.inContaq.utlities.dailysms.DailyTaskParams;
+import nyc.c4q.jonathancolon.inContaq.utlities.sms.Sms;
+
+import static android.graphics.Color.parseColor;
+import static com.db.chart.renderer.AxisRenderer.LabelPosition.NONE;
+
+/**
+ * Created by Hyun on 3/11/17.
+ */
+
+public class DailyGraph{
```

Comment:
this seems to have a lot in common with MonthlyGraph. should also subclass a generic Graph class to hold shared logic